packages/@office-iss/react-native-win32/jest.config.js

```diff
@@ -6,9 +6,16 @@ module.exports = {
     '^.+\\.(bmp|gif|jpg|jpeg|mp4|png|psd|svg|webp)$':
       '<rootDir>/jest/assetFileTransformer.js',
   },
-  transformIgnorePatterns: [],
+  transformIgnorePatterns: ['jest-runner'],
```

Comment:
Was this added by us or by Meta? If us, could we add a note in the PR description on why we needed to add this to get Jest working?

Reply:
We added it to get jest working again! I'll add it to the PR description but it seems like its a jest issue where we shouldn't be transpiliing jest-runner in the first place and this is the work around (https://github.com/facebook/jest/issues/13173)

Reply:
Cool thanks for the context!